src/components/SheetBuilder/SheetBuilderForm/SheetBuilderFormStep/SheetBuilderFormStepOriginDefinition/OriginBenefitsSelect.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {
+  OriginBenefits,
+  SerializedOriginBenefits,
+  Translator,
+} from 't20-sheet-builder';
+import SheetBuilderFormSelect from '../../SheetBuilderFormSelect';
+import { Option } from '../../../common/Option';
+
+export type OriginBenefitOption = Pick<
+  SerializedOriginBenefits,
+  'name' | 'type'
+>;
+
+type Props = {
+  benefits: OriginBenefits;
+  setBenefits(benefits: OriginBenefitOption[]): void;
+};
+
+const OriginBenefitsSelect = ({ benefits, setBenefits }: Props) => {
+  const options: Option<OriginBenefitOption>[] = [];
+
+  benefits.generalPowers.forEach((power) => {
+    options.push({
+      label: Translator.getPowerTranslation(power),
+      value: { type: 'generalPowers', name: power },
+    });
+  });
+
+  benefits.skills.forEach((skill) => {
+    options.push({
+      label: Translator.getSkillTranslation(skill),
+      value: { type: 'skills', name: skill },
+    });
+  });
+
+  options.push({
+    label: Translator.getPowerTranslation(benefits.originPower),
+    value: { type: 'originPower', name: benefits.originPower },
+  });
```

Comment:
Usando um pouco de ecmascript, a gente consegue dar um visual melhor/mais moderno pra esse código, já que ele não tem nenhum modificador muito complexo:
```suggestion
const options: Option<OriginBenefitOption>[] = [
    ...benefits.generalPowers.map((power) => ({
      label: Translator.getPowerTranslation(power),
      value: { type: 'generalPowers', name: power },
    })),
    ...benefits.skills.map((skill) => ({
      label: Translator.getSkillTranslation(skill),
      value: { type: 'skills', name: skill },
    })),
    {
      label: Translator.getPowerTranslation(benefits.originPower),
      value: { type: 'originPower', name: benefits.originPower },
    },
  ];
```

Reply:
melhor msm, nice